calyx-py/calyx/queues.py

```diff
@@ -384,41 +725,66 @@ def __len__(self) -> int:
         return self.pifo_len
 
 
-def operate_queue(queue, max_cmds, commands, values, ranks=None, keepgoing=False):
-    """Given the two lists, one of commands and one of values.
-    Feed these into our queue, and return the answer memory.
+def operate_queue(queue, max_cmds, commands, values, ranks=None, keepgoing=None, times=None):
```

Comment:
Nit, but consider making the `keepgoing` either the last flag or the third, so that it is not in the middle of the four lists

Reply:
I definitely agree with this, it was a poor design decision on my part. Fixed now!